src/include/stir/modelling/PlasmaData.inl

```diff
@@ -55,8 +55,25 @@ void  PlasmaData::read_plasma_data(const std::string input_string)
   if(!data_stream)    
     error("cannot read plasma data from file.\n");    
   else
-    data_stream >> _sample_size ;
-  
+  { 
+    // Get the first line, which should be the number of samples
+    std::string first_line;
+    if (std::getline(data_stream, first_line))
+    {
+      // now first, check if the first line is a single character. 
+      // this is best done in C style, cleaner than iterating over chars
+      char* p;
+      long converted = strtol(first_line.c_str(), &p, 10);
+      if (*p)
```

Comment:
would this not fail if there's trailing whitespace?

Reply:
Yes, I didn't want to add too much checks, but I can certainly improve it if you think its worth it.

Reply:
Well, I'd argue that better to risk having a message saying `error("First line of .if file is not number of samples, check trailing whitespace");` than a segmentation fault in case you did not add the first number as integer. 

I'll add the check.